cmd/raidz_test/raidz_test.h

```diff
@@ -92,13 +92,13 @@ static inline size_t ilog2(size_t a)
 
 #define	LOG(lvl, a...)				\
 {						\
-	if (rto_opts.rto_v >= lvl)		\
+	if ((ssize_t)rto_opts.rto_v >= lvl)		\
```

Comment:
Would it be possible to change the type of the variable itself, instead of casting it here?  Casts are dangerous.

Reply:
Fair cop, moved the verbosity into its own enum and made `rto_v` that enum.

Reply:
Hmmmm, be careful.  That variable is in a struct.  If you expect that struct to have any kind of ABI stability (which you may of course don't care), you should use `int` to represent the `enum` type, since the compiler might change the underlying type as it sees fit.  See a discussion on linux-api@ that starts here: <https://lore.kernel.org/linux-api/8a22a3c2-468c-e96c-6516-22a0f029aa34@gmail.com/>

Reply:
I'm well aware (just as I'm aware that if GCC decided to change how it represents enums all hell would break loose, and 99% of ABIs herein take `boolean_t`s, which are enums); this is an internal API for a two-file program, it only *looks* like it isn't.

Reply:
Also, it seems that you can already do it on Clang and [N2904/N2908](https://twitter.com/__phantomderp/status/1494051757932130306) should bring it to the Other Compiler, so, uh, rejoice?